tests/integration/test_writes/test_writes.py

```diff
@@ -285,6 +285,33 @@ def test_data_files(spark: SparkSession, session_catalog: Catalog, arrow_table_w
     assert [row.deleted_data_files_count for row in rows] == [0, 1, 0, 0, 0]
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize("format_version", [1, 2])
+def test_object_storage_data_files(
```

Comment:
wdyt something like,
```suggestion
def test_object_storage_data_files_paths_hash_entropy(
```

Reply:
Hmm. I slightly prefer `test_object_storage_data_files`. Here's my reasoning: for a test to be called that and check binary directories are present is reasonable (it's what's expected from object storage data files). But the test name specifying that detail reveals to much IMHO.